extensions/sat/json-schema/schema.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "schema.json#",
+  "title": "Sat Extension",
+  "description": "STAC Sat Extension to a STAC Item.",
+  "allOf": [
+    {
+      "$ref": "../../../item-spec/json-schema/item.json#/definitions/core"
+    },
+    {
+      "$ref": "#/definitions/sat"
+    }
+  ],
+  "definitions": {
+    "eo": {
+      "type": "object",
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "properties": {
+          "type": "object",
+          "required": [
```

Comment:
@m-mohr is this sufficient for requiring the fields from the core Item?

Reply:
I think that should work, let'y try it out tomorrow in a validator.
Are these requirements specified in the written spec?